rllib/utils/minibatch_utils.py

```diff
@@ -84,14 +102,10 @@ def __iter__(self):
                 # concatenate all the samples, we should have minibatch_size of sample
                 # after this step
                 minibatch[module_id] = concat_samples(samples_to_concat)
-                # roll miniback to zero when we reach the end of the batch
+                # roll minibatch to zero when we reach the end of the batch
                 self._start[module_id] = e
 
-            # TODO (Kourosh): len(batch) is not correct here. However it's also not
-            # clear what the correct value should be. Since training does not depend on
-            # this it will be fine for now.
-            length = max(len(b) for b in minibatch.values())
-            minibatch = MultiAgentBatch(minibatch, length)
+            minibatch = MultiAgentBatch(minibatch, 0)
```

Comment:
The length of the batch is not needed in the Learner.
We should get rid of MultiAgentBatch here anyway, there is no reason to use anything more than a dict.